rankers/SimpleAggregateRanker/tests/test_aggregate_ranker.py

```diff
@@ -0,0 +1,274 @@
+import numpy as np
+
```

Comment:
would it be possible to leverage `pytest` parametrization for these tests?

Reply:
sure, I updated the code to use parametrization.